### PR TITLE
Remove DO_SET_SERVO support & comment cleanup

### PR DESCRIPTION
**Pull request by junwoo091400**
## Describe problem solved by this pull request
This removes the support for [DO_SET_SERVO](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_SERVO) MAVLink command.

Existing problems:
- DO_SET_SERVO is redundant to DO_SET_ACTUATOR
- DO_SET_SERVO was getting treated as DO_SET_ACTUATOR in `mission_feasiblity_checker.cpp` wrongly

Also, with the support of Control allocation using normalized command values, commanding PWM values is quite obsolete. This would need to be done in a better way.

## Describe your solution
- Remove DO_SET_SERVO enums from the codebase
- Unify DO_SET_SERVO handling into only DO_SET_ACTUATOR

## Describe possible alternatives
Setting Servo (as one of control allocator's function set) value directly would be the obvious alternative, but I thought that's a bad idea because:

For example, when a fixed wing is flying and you command DO_SET_SERVO, do you really want your Servo to freeze in one position?

Therefore, I think this should rather be handled in Offboard Actuators, to decouple flight control core controls / auxilary actuators, servos

## Additional context
Related PR: https://github.com/PX4/PX4-Autopilot/pull/20325

### CODE REVIEW

File: src/modules/mavlink/mavlink_mission.cpp [hunk 1]
```diff
@@ -1331,7 +1331,7 @@ MavlinkMissionManager::parse_mavlink_mission_item(const mavlink_mission_item_t *
 	    mavlink_mission_item->frame == MAV_FRAME_GLOBAL_RELATIVE_ALT ||
 	    (_int_mode && (mavlink_mission_item->frame == MAV_FRAME_GLOBAL_INT ||
 			   mavlink_mission_item->frame == MAV_FRAME_GLOBAL_RELATIVE_ALT_INT))) {
-		// This is a mission item with a global coordinate
+		/** Mission item with a global coordinate **/
```
Comment (bkueng) [inline src/modules/mavlink/mavlink_mission.cpp:1334]: Can you move unrelated changes into a separate commit? And please don't add another type of comments.
```suggestion
		// Mission item with a global coordinate
```

File: src/modules/navigator/mission_feasibility_checker.cpp [hunk 2]
```diff
@@ -290,7 +289,7 @@ MissionFeasibilityChecker::checkMissionItemValidity(const mission_s &mission)
 		}
 
 		/* Check non navigation item */
-		if (missionitem.nav_cmd == NAV_CMD_DO_SET_SERVO) {
+		if (missionitem.nav_cmd == NAV_CMD_DO_SET_ACTUATOR) {
```
Comment (bkueng) [inline src/modules/navigator/mission_feasibility_checker.cpp:292]: This isn't correct. Can you just remove the block?

File: src/modules/mavlink/mavlink_mission.cpp (hunk 1 above)
Comment (junwoo091400) [reply src/modules/mavlink/mavlink_mission.cpp:1334]: I wanted to emphasize the section by having /**/ comment, since it is a very long if statement.
What is the downside of having this type of comment?

Comment (bkueng) [reply src/modules/mavlink/mavlink_mission.cpp:1334]: It's doxygen-style, so used for a different purpose. Plus imo such somewhat random formattings don't help with consistency. In most places we're moving to single-line comments (`//`).

**Comment by sfuhrer** [comment]
Can we get this over the finish line? Would solve one open issue I had to remove actuator_controls in https://github.com/PX4/PX4-Autopilot/pull/20837.

**Comment by junwoo091400** [comment]
> Can we get this over the finish line?

Sorry I didn't get to check on this before, how did you solve the issue in your PR that got merged?

**Comment by sfuhrer** [comment]
> > Can we get this over the finish line?
> 
> Sorry I didn't get to check on this before, how did you solve the issue in your PR that got merged?

I removed it in https://github.com/PX4/PX4-Autopilot/pull/20837/commits/1cde9a5adc188fc2c25354e646c1b8ec8c82ad73, as it was redundant as you also stated.
We then can close this PR guess?
